cursorless-talon/src/modifiers/containing_scope.py

```diff
@@ -43,6 +43,15 @@
     "tags": "xmlBothTags",
     "start tag": "xmlStartTag",
     "end tag": "xmlEndTag",
+    # LaTeX
+    "command": "latexCommand",
```

Comment:
As mentioned elsewhere, I think I'd be inclined to remove this one and just use `functionCall`